sql/sql_insert.cc

```diff
@@ -474,7 +474,7 @@ void upgrade_lock_type(THD *thd, thr_lock_type *lock_type,
     }
 
     bool log_on= (thd->variables.option_bits & OPTION_BIN_LOG);
-    if (thd->wsrep_binlog_format() == BINLOG_FORMAT_STMT &&
+    if (WSREP_BINLOG_FORMAT(global_system_variables.binlog_format) == BINLOG_FORMAT_STMT &&
```

Comment:
The length of the source code line is 91 columns, which exceeds the maximum by 12 columns. I would suggest to split the line after the `==`.

I ran `binlog.binlog_unsafe` with this fix on 10.5 with MemorySanitizer, and the test passed.

Why is the code not being enclosed in `#ifdef WITH_WSREP`?

Reply:
Because original code was global_system_variables.binlog_format == BINLOG_FORMAT_STMT before wsrep,
macro used will use either global_system_variables.binlog_format or in wsrep case wsrep_forced_binlog_format if set